tests/functional/utils.py

```diff
@@ -64,3 +65,20 @@ def recreate_censored_content(content: str, matched_string: str) -> str:
     """Applies ggshield censoring to any occurrence of `matched_string` inside
     `content`. Returns the censored string."""
     return content.replace(matched_string, recreate_censored_string(matched_string))
+
+
+# Use this in iac tests until iac hooks are added to the install command
+# Meaning they are added to install.py
+def create_local_hook(
+    path: Path,
```

Comment:
Directly specifying the hook dir path would allow to reuse this function for pre-receive also.
In the pre-receive context, the hook is executed in the `.git/` directory.